OpenRA.Mods.Common/Traits/Player/ProductionQueue.cs

```diff
@@ -68,15 +68,17 @@ public class ProductionQueue : IResolveOrder, ITick, ITechTreeElement, INotifyOw
 		public readonly ProductionQueueInfo Info;
 		readonly Actor self;
 
+		// A list of things we could possibly build
+		readonly Dictionary<ActorInfo, ProductionState> produceable = new Dictionary<ActorInfo, ProductionState>();
+		readonly List<ProductionItem> queue = new List<ProductionItem>();
+		readonly IEnumerable<ActorInfo> allProducables;
+		readonly IEnumerable<ActorInfo> buildableProducables;
```

Comment:
Oddly I went for the correct spelling of producible initially but decided to change it back rather than introduce inconsistency/change everything else. Instead I managed to spell it wrong but differently to the rest of the wrong spellings. Heh.